tasks

```diff
@@ -124,24 +124,36 @@ const DojoVersion = \"${next_version}\"
         ;;
     homebrew_tap)
         if [ ! -f bin/dojo_darwin_amd64 ]; then echo "dojo_darwin_amd64 binary does not exist"; exit 1; fi
-        SHA256=$(sha256sum bin/dojo_darwin_amd64 | awk '{printf $1}')
+        if [ ! -f bin/dojo_darwin_amd64 ]; then echo "dojo_linux_amd64 binary does not exist"; exit 1; fi
```

Comment:
I think this should be `bin/dojo_linux_amd64`

Reply:
Good catch! I'll update it.